.env.sample

```diff
@@ -15,4 +15,5 @@ WEB3_NODE_URL_137=https://infura.io
 REFERRAL_DELIMITER_START_TIMESTAMP=1657290720
 ENABLE_SPOKE_POOLS_EVENTS_PROCESSING=true
 ENABLE_REFERRALS_MATERIALIZED_VIEW_REFRESH=true
+STICKY_REFERRAL_ADDRESSES_MECHANISM=queue
```

Comment:
`queue | cron | disabled` are the possible values here

Reply:
do you think its worth commenting that these are possible values?

Reply:
Good point. Added comment [here](https://github.com/across-protocol/scraper-api/commit/30c6413d3ddef4a967bc4ed4e6a7206f1cc18538)